src/routes/results/[results_string]/+page.svelte

```diff
@@ -59,6 +60,7 @@
 
 	function openModal(): void {
 		showEmailModal = true;
+		trackEvent('click_send_email');
```

Comment:
nit: maybe since this is just the modal opening, for clarity we could say "click_email_results_button",  and if we want to additionally track the send email button we could add this name there?